crates/network-primitives/src/traits.rs

```diff
@@ -51,6 +51,14 @@ pub trait ReceiptResponse {
 
     /// EIP-7702 Authorization list.
     fn authorization_list(&self) -> Option<&[SignedAuthorization]>;
+
+    /// Returns the cumulative gas used at this receipt.
+    fn cumulative_gas_used(&self) -> u128;
+
+    /// The post-transaction state root (pre Byzantium)
+    ///
+    /// EIP98 makes this field optional.
+    fn state_root(&self) -> Option<&B256>;
```

Comment:
we can ingore this because no longer useful

Reply:
when converting from `ReceiptResponse` to `TransactionReceipt<T>`, would it not be a bug to assume the field to be `None` for historic blocks?

Reply:
> when converting from ReceiptResponse to TransactionReceipt<T>
this isn't the purpose of this trait, this trait just serves the purpose of providing expected values

but I guess it doesn't really hurt to add this function, even if this is useless.